Content.Server/Power/EntitySystems/CableMultitoolSystem.cs

```diff
@@ -28,31 +30,39 @@ public sealed class CableMultitoolSystem : EntitySystem
     {
         [Dependency] private readonly ToolSystem _toolSystem = default!;
         [Dependency] private readonly PowerNetSystem _pnSystem = default!;
+        [Dependency] private readonly ExamineSystemShared _examineSystem = default!;
 
         public override void Initialize()
         {
             base.Initialize();
 
-            SubscribeLocalEvent<CableComponent, ExaminedEvent>(OnExamined);
+            SubscribeLocalEvent<CableComponent, GetExamineVerbsEvent>(OnGetExamineVerbs);
         }
 
-        private void OnExamined(EntityUid uid, CableComponent component, ExaminedEvent args)
+        private void OnGetExamineVerbs(EntityUid uid, CableComponent component, GetExamineVerbsEvent args)
```

Comment:
Maybe its worth supporting disabled verbs & verb tooltips. Then you could do something like having cables show a disabled power-examine button that shows a "requires multitool" tooltip on hovering over it?

That would make it obvious that there is more to see on an entity & how to go about getting that info.

Reply:
ooh, i like that